README.md

```diff
@@ -1 +1,23 @@
-# Algorithm-Visualization
\ No newline at end of file
+# Algorithm-Visualization - AlgViz
+
+Make SVGs of data structures from your code.
+
+## Development
+
+This package is supported by [pipenv](https://github.com/pypa/pipenv) for development.
+
+#### Setup
+
+```
+pipenv install
+pipenv shell
+```
+
+#### Linting
+We rely on pipenv's built in Flake 8 code style checker.
```

Comment:
We should keep separate documentation for users and for developers.  How about this goes in `DEVELOPMENT.md`?